app/views/posts/_posts.html.erb

```diff
@@ -0,0 +1,27 @@
+
+<% @posts.each do |post| %>
+      <div>
+       <% if member_signed_in? %>
+          <% if post.member %>
+            <h3><%= post.member.name %></h3>
+          <% end %>
+        <%end%>
+         <h1><%= post.title %></h1>
+        <p><%= post.body %></p>
+        <% if member_signed_in? %>
+          <nav class="navbar">
+            <div class="nav navbar-nav ml-auto">
+              <%= link_to post do %>
+                <span class="icon"><i class="fa fa-link" aria-hidden="true"></i></span>
+              <% end %>
+              <%= link_to edit_post_path(post) do %>
+                <span class="icon"><i class="fa fa-pencil" aria-hidden="true"></i></span>
+              <% end %>
+              <%= link_to post, method: :delete, data: { confirm: 'Are you sure?' } do %>
+                  <span class="icon"><i class="fa fa-trash-o" aria-hidden="true"></i></span>
+                <% end %>
+            </div>
+          </nav>
+        <% end %>
```

Comment:
## Remove unnecessary code:

- [ ]  Since we are restricting out `RESTFUL` API in the [config/routes](https://github.com/mihrab34/members-only/edit/member-post/config/routes.rb) with this the following lines of code 👇 

  `  resources :posts, only: [:new, :create, :index]`, we won't be needing this implementation as it causes the following error 👇 Kindly work on this.

![Screenshot 2020-09-30 015527](https://user-images.githubusercontent.com/22345961/94626040-de372900-02c2-11eb-92bf-c5cb5e9c317a.png)